100_Numpy_exercises_with_hints_with_solutions.md

```diff
@@ -409,12 +409,12 @@ Z.flags.writeable = False
 Z[0] = 1
 ```
 #### 44. Consider a random 10x2 matrix representing cartesian coordinates, convert them to polar coordinates (★★☆)
-`hint: np.sqrt, np.arctan2`
+`hint: np.hypot, np.arctan2`
 
 ```python
 Z = np.random.random((10,2))
 X,Y = Z[:,0], Z[:,1]
-R = np.sqrt(X**2+Y**2)
+R = np.hypot(Y,X)
```

Comment:
Y,X -> X,Y (for consistency)

Reply:
The order is such so that it's consistent with the order of arguments in the `arctan2` invocation. It's my personal mnemonic, but I will change it if you insist.

Reply:
I would prefer yes.